main/tables.py

```diff
@@ -151,6 +151,10 @@ def _style_fraction(self, value: tuple[int, float]) -> str:
 class FundingTable(tables.Table):
     """Table for the Funding listing."""
 
+    funding_body = tables.Column(
+        linkify=("main:funding_detail", {"pk": tables.A("pk")})
+    )
+
```

Comment:
The funding body should be in the table (I thought it was already there, to be honest) but should not be linked to the funding detail. For that is the link in the `project_code`. Instead, add a description as done, for example, in `effort`:

```suggestion
    funding_body = tables.Column(
        attrs={
            "th": {
                "title": "Entity providing this funding, if external."
            }
        },
    )

```